frontend/src/pages/audits/edit/general/general.js

```diff
@@ -296,6 +295,45 @@ export default {
                 const needle = Utils.normalizeString(val)
                 this.selectCompanies = this.companies.filter(v => Utils.normalizeString(v.name).indexOf(needle) > -1)
             })
+        },
+
+        toggleAskReview: function() {
+            AuditService.updateAuditGeneral(this.auditId, { isReadyForReview: !this.audit.isReadyForReview })
+            .then(() => {
+                this.$emit('toggleAskReview');
+                this.audit.isReadyForReview = !this.audit.isReadyForReview;
+                this.auditOrig.isReadyForReview = this.audit.isReadyForReview;
+                Notify.create({
+                    message: 'Audit review status updated successfully',
+                    color: 'positive',
+                    textColor:'white',
+                    position: 'top-right'
+                })
+            })
+            .catch((err) => {             
+                console.log(err.response)
+            });
+        },
+
+        toggleApproval: function() {
+            AuditService.toggleApproval(this.auditId)
+            .then(() => {
+                this.$emit('toggleApproval');
+
+                Notify.create({
+                    message: 'Audit approval updated successfully',
+                    color: 'positive',
+                    textColor:'white',
+                    position: 'top-right'
+                })
+            })
+            .catch((err) => {          
+                console.log(err.response)
+            });
+        },
```

Comment:
I dont think this is necessary anymore? I think they are now in index.vue of the parent? If you made changes, make sure that they are reflected in the new function, and that you do not use these functions anywhere

Reply:
Right, my bad. Thought I was using them in the icon, but I am not!